scripts/start.sh

```diff
@@ -6,7 +6,7 @@ set - o ipfs shutdown kill
 echo "Running: "$0
 
 mongod --logpath /dev/null &
-jsipfs daemon &
+node ./scripts/startIpfs &
```

Comment:
Now that this is a node script, i think we should take this logic out of shell and put it into the `start.js` script, so we can start cleaning these scripts up. Eventually it would be great to not have to use any shell scripting at all. 

You could probably write the `startIpfs` script a lot like `compile`: https://github.com/openworklabs/quasar/blob/primary/scripts/compileAndMigrate.js#L5